### ISSUE TITLE
mwa_client doesn't accept 0 as a valid choice for edgewidth

### ISSUE THREAD

**Issue by johnsmorgan** [MWATelescope/manta-ray-client#26]
0 is a valid choice for edge flagging and should be treated as such (even if it should not be the default)

**Comment by adityajnhs** [comment]
We submitted a job with "0" edgewidth, but we are getting the following errors:

manta-ray-client version 1.0.0
Connecting to MWA ASVO (asvo.mwatelescope.org:8778)...
Connected to MWA ASVO
Error submitting job #1 from csvfile. Details below:
{"error_code": 0, "error": "Edge Width: Expected > 0"}


We have conducted observations to search for HI 21 cm line in absorption (project D0027). We will lose the line data if we flag 80 kHz channels in each coarse channel of 1.28 MHz, in the 30.7 MHz band. So, we like to opt for "0" kHz edge channel.   

We request you to look into this issue.
